dspace-server-webapp/src/main/java/org/dspace/app/rest/submit/step/validation/MetadataValidation.java

```diff
@@ -96,6 +98,14 @@ public List<ErrorRest> validate(SubmissionService submissionService, InProgressS
                             "/" + WorkspaceItemRestRepository.OPERATION_PATH_SECTIONS + "/" + config.getId() + "/" +
                                 input.getFieldName());
                     }
+                    if ("local.hasCMDI".equals(fieldName)) {
+                        try {
+                            Context context = ContextUtil.obtainCurrentRequestContext();
+                            CMDIFileBundleMaintainer.updateCMDIFileBundle(context, obj.getItem(), mdv);
+                        } catch (Exception e) {
+                            e.printStackTrace();
```

Comment:
Is this really used also elsewhere?

Reply:
No, but for testing purposes I separated it.